### ISSUE TITLE
 Extend RTOFS GEMPAK files from 72 to 144 hours and to all regions (bugzilla #657)

### ISSUE THREAD

**Issue by DanIredell-NOAA** [NOAA-EMC/RTOFS_GLO#5]
http://www2.spa.ncep.noaa.gov/bugzilla/show_bug.cgi?id=657

These are OPC scripts and they need to specify their requirements.  Currently there are these scripts:
ush/rtofs_surface_day1-3.sh
ush/rtofs_surface_day4-5.sh
ush/rtofs_surface_hcasts.sh

Description:

With the new Global RTOFS upgrade we started getting GEMPAK for the Arctic, Alaska and Bering sea files. These are available out to 72 hours. There are Grib2 files with forecasts out to 144 hours at a 3 hourly time step named, rtofs_glo.t00z.f144*grib2 for the 4 regions shown below. These get made later in the day. Could you please add these later forecasts to our GEMPAK? This is not a rush but would be helpful to have the full forecasts.
Source Grib2 files for rtofs:

rtofs_glo.t00z.f072_alaska_std.grb2
rtofs_glo.t00z.f144_alaska_std.grb2
rtofs_glo.t00z.f072_arctic_std.grb2
rtofs_glo.t00z.f144_arctic_std.grb2
rtofs_glo.t00z.f072_bering_std.grb2
rtofs_glo.t00z.f144_bering_std.grb2
rtofs_glo.t00z.f072_west_atl_std.grb2
rtofs_glo.t00z.f144_west_atl_std.grb2

For the West Atlantic files, could you please add the West Atlantic files to the NCO GEMPAK production?
There are files in $MODEL for West Atlantic but these are getting made by OPC and we dbnet them now.

--- further correspondence from OPC ---
"while they would like it to be extended beyond 72 hours, they are OK with the current system. They don't have to change anything with the current setup.". It may be moving from gempak to AWIPS in OPC, 
Therefore we will leave it as it is, and will not extend beyond 72 hours in this upgrade (rtofs.v2.0). It will be revisited in the next upgrade to see if it is necessary to extend. Leave this ticket to open for future revisit.

**Comment by DanIredell-NOAA** [comment]
correspondence from OPC in January 2024.  (Bob Daniels)

_We will not need more GEMPAK files as we are transitioning to AWIPS. 
With help from Logan Dawson, Todd and Deanna and I looked at RTOFS in AWIPS. 
We found helpful grids for OPC forecasting including regions covering West Conus, West Atlantic, and Alaska. Sea surface temperature, and surface currents were available. 
We seem to have what we need now as far as coverage and grids._
